nncf/common/tensor_statistics/aggregator.py

```diff
@@ -23,6 +23,7 @@
 from nncf.data.dataset import DataItem
 from nncf.data.dataset import Dataset
 from nncf.data.dataset import ModelInput
+from nncf.common.logging import nncf_logger
```

Comment:
linter: move this line after
 `from nncf.common.graph.transformations.layout import TransformationLayout`